change/@ni-nimble-components-e476eedb-989a-4246-811b-039ca8a8352e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Actually I don't think I see any tests around nested lists. I was able to create a nested list using tab / shift +tab.

Is this feature intended to be enabled? If so we should probably have buttons to increase / decrease indentation and address the issue shown above with the button state in nested lists. Also some tests in editor and viewer.

If not maybe something needs to be done to disable nested lists?

Reply:
Oh must be intentional, it's in the Angular app demo 🤣
![image](https://github.com/ni/nimble/assets/1588923/bbcc3a82-c67c-467c-91f3-674232bcec38)
So yea seems like there are some behaviors to double check as ddescribed

Reply:
Yes, this is intentional and the behavior seen in enabling the state of both the list buttons is the default behavior in Tiptap. Here is the demo shown in Tiptap to indicate its state.
![nested-list-tiptap](https://github.com/ni/nimble/assets/123377523/12d4f125-c97f-4ecd-9f4f-d6a3564e42a7)

Tiptap states that this is an expected behavior: https://github.com/ueberdosis/tiptap/issues/2475#issuecomment-1028813976 and if needed we need to write our own implementation to handle this case. Here is the reference to get the active state of the separate list nodes: https://github.com/ueberdosis/tiptap/issues/1058#issuecomment-778254557

For now, I couldn't find a way to disable the nested list in Tiptap. However, I have written test cases for the nested lists in the editor and viewer with the current implementation. 

@vikisekarNI @suseendran-ni Could you please share your inputs here on whether the current behavior is expected or to have the special handling for the nested lists active state? Also, should we disable the nested list if possible?

Reply:
I was able to reproduce this behavior and makes sense to have only the last active node's list button highlighted (bullet/numbered list).

@rajsite - We need to do some more research on how much effort it would cost. So will it be fine, if we can track this in another work item under same feature and push it in subsequent PRs?

Reply:
Thanks for the tests for the current behavior 👍

Making a separate issue to track it makes sense. And communicating the work and cost of maintaining a change to tiptap's behavior.

Reply:
Created a new issue #1434. Kindly advise on issue type suggestion/cross-linking if any.

Reply:
A user story makes sense. You can add it to the task list in the description of #1288 to create a link between them.

Reply:
I have added this to a checklist of https://github.com/ni/nimble/issues/1288. Therefore, I am closing this comment and will be creating a separate PR to address this issue.